README.md

```diff
@@ -158,23 +145,21 @@ The script spins up 5 local drand nodes using Docker and produces fresh
 randomness every 10 seconds.
 
```

Comment:
This is just making me realize we should add a link to the readme of the demo in the folder `demo/` (it explains how to query the nodes as well)

Reply:
Added.